### PR TITLE
update async guide to add async opertaions

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Add other async operations to the async guide

### CODE REVIEW

File: learn/advanced/asynchronous_operations.md [hunk 1]
```diff
@@ -17,12 +17,16 @@ Currently, these are Meilisearch's asynchronous operations:
...
+- Swapping indexes
 - Deleting an index
 - Updating index settings
 - Adding documents to an index
 - Updating documents in an index
 - Deleting documents from an index
+- Canceling a task
+- Deleting a task
 - Creating a dump
+- Creating snapshots
```
Comment (dichotommy) [inline learn/advanced/asynchronous_operations.md:29]: Creating snapshots is a task? I'm not so sure about that. There's no endpoint to call to create a snapshot; they are scheduled regularly.

Comment (maryamsulemani97) [reply learn/advanced/asynchronous_operations.md:29]: Isn't it though? https://docs.meilisearch.com/reference/api/tasks.html#snapshotcreation

Comment (guimachiavelli) [reply learn/advanced/asynchronous_operations.md:29]: I believe @maryamsulemani97 is correct and snapshots is a task. There are at least [two](https://github.com/meilisearch/documentation/pull/2044/files) [places](https://github.com/meilisearch/specifications/blob/b2e4b3df55574f916f8e023c8d558f651e49471e/open-api.yaml#L345) in the specs saying creating snapshots generates a task.

**Review by guimachiavelli — APPROVED** [review]
🦺

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**